plugins/home/src/homePageComponents/Banner/Banner.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import classNames from 'classnames';
+import { BackstageTheme } from '@backstage/theme';
+import { makeStyles } from '@material-ui/core/styles';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import { DateTime } from 'luxon';
+
+type BannerProps = {
+  startTime: string;
+  endTime: string;
+  info: React.ReactNode;
+  action?: React.ReactNode;
+  image?: string;
+  isOpen?: boolean;
```

Comment:
added this isOpen optional prop to be able to close it with a passed in action

Reply:
Would be nice if the close action could be optional, an idea could follow the same interface as the Material UI [Alert](https://v4.mui.com/api/alert/) component:
- If an onClose handler is passed, show the close button.
- if the action component is passed, replace the close button with the action component.

Here is an [example](https://v4.mui.com/components/alert/#actions).

Reply:
+1, would also say that if we need to keep this prop I think it should be `open` rather than `isOpen`, boolean props don't need an `is` prefix